resources/docs/accounts.md

```diff
@@ -0,0 +1,90 @@
+3. **Test** the connection before proceeding. If there are issues you will see an error message appear.
+4. Click **OK**.
+
+RSS Guard will start syncing your feeds.
+
+## Inoreader
+
+Before using this integration it is highly recommended you configure your own Inoreader client application according to [Inoreader documentation](https://www.inoreader.com/developers/register-app); when setting this up you _must_ configure the **Redirect URI** to `http://localhost` and the **Scope** to Read and write. Once this is done you can set it up in RSS Guard:
+
+1. Go to **Accounts → Add New Account**. Select **Inoreader**.
+2. Enter the required information:
+    - Username: As entered to log in to Inoreader.
+    - Application ID: As provided by Inoreader.
+    - Application Key: As provided by Inoreader.
+    - Redirect URL: `http://localhost`, as entered in the Inoreader OAuth 2.0 configuration page.
+3. Click **Login**. This should open a browser window with an Inoreader authorization page.
+4. Click **Authorize** and close the browser window.
+5. In RSS Guard click **OK** to confirm the account addition.
+
+RSS Guard will start syncing your feeds.
+
+## Nextcloud News
+
+[Nextcloud News](https://github.com/nextcloud/news) is an installable app that lets you view RSS within Nextcloud. RSS Guard can use it store the list of feeds and manipulate their state.
+
+To set it up:
+
+1. Go to **Accounts → Add New Account**. Select **Nextcloud News**.
+2. Enter the required information:
+    - URL: `https://nextcloud.server.example`. Do not enter any API or WebDAV information. RSS Guard will look for it automatically.
+    - Download only unread messages
+    - Force execution of server-side update when updating feeds from RSS Guard forces Nextcloud to refresh all the feeds stored. Depending on how many system resources are available to Nextcloud this _will_ cause you issues.
+    - Limit number of downloaded messages per feed. Limiting this is useful if you're on a mobile or rate-limited connection.
+    - Username: As it would be to log in to your Nextcloud instance.
+    - Password
+
+The minimum supported version of Nextcloud News is 6.0.5; any version of Nextcloud released in the last year should be compatible.
+
+## Standard online feeds (RSS/RDF/ATOM)
+
+This account type lets RSS Guard function as a stand-alone RSS client. It saves feeds and their state to the database and does not sync it with any external servers or applications. This _can_ be useful if you wish to sync the entire RSS Guard database if you do not plan on using any of the available integrations. Be aware the database can grow to huge sizes and you will see increased network traffic as your syncing application will have to keep up with the manipulation of feeds within RSS Guard itself.
+
+1. Go to **Accounts → Add New Account**. Select Standard online feeds (RSS/RDF/ATOM)
```

Comment:
The set of "predefined" feeds is only there so that user can choose it so that there is something to the start. Really do not document what exact feeds are there because the predefined feed sets can be customized by "country". For example user in Czech republic could see other feeds than "english" user. The list for english user is here: https://github.com/martinrotter/rssguard/tree/master/resources/initial_feeds (feel free to PR and add some more relevant feeds, I plan to add list for Czech users too in the future).